README.md

```diff
@@ -1,33 +1,31 @@
-# mesa-geo - a GIS extension to Mesa Agent-Based Modeling
+# Mesa-Geo: a GIS extension for the Mesa agent-based modeling framework in Python
```

Comment:
I was initially wondering that mesa-geo is a deliberate lower case project name, along the line of macOS (not MacOS) and nginx (or NGINX, but not Nginx). I suppose Mesa-Geo is more consistent with Mesa itself.